purchase_isolated_quotation/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Ecosoft (<http://www.ecosoft.co.th>)
```

Comment:
@phongyanon I think we can remove the copyright here. Already have the contributor list in README